packages/@aws-cdk/aws-applicationautoscaling/lib/target-tracking-scaling-policy.ts

```diff
@@ -186,8 +186,14 @@ export enum PredefinedMetric {
   /**
    * DYANMODB_WRITE_CAPACITY_UTILIZATION
    * @see https://docs.aws.amazon.com/autoscaling/application/APIReference/API_PredefinedMetricSpecification.html
+   * @deprecated use `PredefinedMetric.DYNAMODB_WRITE_CAPACITY_UTILIZATION`
    */
   DYANMODB_WRITE_CAPACITY_UTILIZATION = 'DynamoDBWriteCapacityUtilization',
+  /**
+   * DYNAMODB_WRITE_CAPACITY_UTILIZATION
+   * @see https://docs.aws.amazon.com/autoscaling/application/APIReference/API_PredefinedMetricSpecification.html
+   */
+  DYNAMODB_WRITE_CAPACITY_UTILIZATION = 'DynamoDBWriteCapacityUtilization',
```

Comment:
The new property needs to be above the deprecated property (see https://github.com/aws/aws-cdk/pull/17731 for details).

Reply:
Changed as proposed.

Reply:
@comcalvi Do you have any ideas how I could fix the build error? Value `DYANMODB_WRITE_CAPACITY_UTILIZATION` was not removed. Is there something wrong with the validation? Thanks.

```
@aws-cdk/aws-applicationautoscaling... CHANGES.
Original assembly: @aws-cdk/aws-applicationautoscaling@1.137.0
Updated assembly:  @aws-cdk/aws-applicationautoscaling@0.0.0
API elements with incompatible changes:
err  - ENUM VALUE @aws-cdk/aws-applicationautoscaling.PredefinedMetric.DYANMODB_WRITE_CAPACITY_UTILIZATION: member DYANMODB_WRITE_CAPACITY_UTILIZATION has been removed [removed:@aws-cdk/aws-applicationautoscaling.PredefinedMetric.DYANMODB_WRITE_CAPACITY_UTILIZATION]
```

Reply:
@jumic I was mistaken about the effects of the above-mentioned jsii bug. It turns out that because only the first enum member is accepted, the second one is entirely removed from the jsii assembly, which is equivalent to deleting it; this results in a breaking change. 

The only way to truly fix this is to give the new enum a dummy value that will be replaced by the correct value whenever it's used. We can do this as follows:

1. Change the value of `DYNAMODB_WRITE_CAPACITY_UTILIZATION`  to a dummy value, something like `'DynamoDBWriteCapacityUtilization-dummy'`.
2. Anytime the props are used to assign a value (pretty sure [here](https://github.com/aws/aws-cdk/blob/6f318b5d8fa4d7a2ada79ceb78d4d26b40a93720/packages/%40aws-cdk/aws-applicationautoscaling/lib/target-tracking-scaling-policy.ts#L146-L149) is the only spot), add a check (`if` statement or ternary operator, preferably) to find the dummy value and replace it with the correct value (which `DYANMODB_WRITE_CAPACITY_UTILIZATION` will have).

Would you be willing to make those changes? It's totally fine if you're not, this is a good bit more work than this change should be.

Reply:
@comcalvi Thanks, this fixed the build.
I also added a test case for this special logic. I hope this is okay.